src/app/store/reboot-info/reboot-info.actions.ts

```diff
@@ -0,0 +1,10 @@
+import { createAction, props } from '@ngrx/store';
+import { SystemRebootInfo } from 'app/interfaces/reboot-info.interface';
+
+export const rebootInfoLoaded = createAction(
+  '[Reboot Info API] Reboot Info Loaded',
+  props<{
+    thisNodeInfo: SystemRebootInfo | null;
```

Comment:
These keys should have `reboot` keyword in there somewhere. They are not node-info. They are specifically node-reboot-info. So, just calling them `thisNodeInfo` and `otherNodeInfo` can cause confusion in the future.